extensions/cactus-plugin-htlc-coordinator/src/test/typescript/unit/api-surface.ts

```diff
@@ -0,0 +1,9 @@
+const tap = require("tap");
```

Comment:
@jagpreetsinghsasan Same request here as for the other api-surface.ts file in this PR.